### DISCUSSION TITLE
Text Field

### DISCUSSION THREAD

**Discussion by mahdi81007** [mantinedev/mantine#6420] [Q&A]
please add Text Field input

**Comment by rtivital** [comment]
https://mantine.dev/core/text-input/

**Comment by mahdi81007** [comment]
now i mean input like this :
![unnamed](https://github.com/mantinedev/mantine/assets/109016246/12fca235-3bf6-4ca6-82db-f14754060a71)

**Reply by rtivital** [reply]
https://mantine.dev/core/input/#inputbase-component
